doc/website/concepts/architecture.md

```diff
@@ -55,7 +55,7 @@ The module `iceoryx_binding_c` makes the inter-process communication features of
 The package `iceoryx_dds` provides a bi-directional DDS gateway using [Eclipse Cyclone DDS](https://cyclonedds.io/).
 The gateway can be used to send data over a network e.g. via Ethernet.
 
-For more information, refer to the [Readme](../../../iceoryx_dds/README.md).
+For more information, refer to the [Readme](https://github.com/eclipse-iceoryx/iceoryx-gateway-dds/blob/main/README.md).
```

Comment:
Btw, do you plan to merge the PR in iceoryx-gateway-dds first to make the like checker happy? Maybe by creating a dummy README.md?

Reply:
@elBoberido Nah, I think that would be way too much effort for not a lot of gain. I think we're ready to merge this PR and finish https://github.com/eclipse-iceoryx/iceoryx-gateway-dds/pull/1 off asynchronously.

Reply:
How about changing the link to `https://github.com/eclipse-iceoryx/iceoryx-gateway-dds/` and adding an action item to the other PR to update this file? Currently this is a dead link and we do not know how long it will take to merge the other PR.